spec/spec_helper.rb

```diff
@@ -32,9 +33,6 @@
 Capybara.default_driver = :rack_test      # This is a faster driver
 Capybara.javascript_driver = :poltergeist # This is slower
 Capybara.default_wait_time = ENV['TRAVIS'] ? 30 : 15
-# HttpLogger.logger = Logger.new(STDOUT)
-# HttpLogger.ignore = [/localhost:8983\/solr/]
-# HttpLogger.colorize = false
```

Comment:
I think leaving these here are helpful for debugging. And show developers how they could do it. You just don't want them activated.